vignettes/why-it-works.Rmd

```diff
@@ -0,0 +1,191 @@
+
+### Uniform primary event time ($r=0$)
+
+In this case, the primary event time is uniformly distributed within the primary censoring window. The survival function of $S_+$ is then:
+
+$$
+Q_{S_+}(t) = Q_T(t + W_P) + { 1 \over W_P} \int_0^{W_P} f_T(t+p) p~ dp.
+$$
+This is analytically solvable whenever the upper distribution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.
+
+In each case it is easier to change the integration variable:
+
+$$
+\begin{aligned}
+Q_{S_+}(t) &= Q_T(t + W_P) + { 1 \over W_P} \int_t^{t+W_P} f_T(z) (z-t)~ dz \\
+&= Q_T(t + W_P) + { 1 \over W_P} \Big[  \int_t^{t+W_P} f_T(z) z~ dz - t \Delta_{W_P}F_T(t) \Big].
+\end{aligned}
+$$
+
+
+#### Gamma distribution
+
+The Gamma distribution has the density function:
+
+$$
+f_T(z;k, \theta) = {1 \over \Gamma(k) \theta^k} z^{k-1} \exp(-z/\theta).
+$$
+
+This gives a solution to the survival function of $S_+$ in terms of analytically available functions:
+
+$$
+\begin{aligned}
+Q_{S_+}(t; k, \theta) &= Q_T(t + W_P; k, \theta) + { 1 \over W_P} \left( {1 \over \Gamma(k) \theta^k} \int_t^{t+W_P} z^{k-1}  \exp(-z/\theta) (z-t)~ dz \right), \\
+&= Q_T(t + W_P; k, \theta) + { 1 \over W_P} \big[ k \theta \Delta_{W_P}F_T(t; k+1, \theta) - t \Delta_{W_P}F_T(t; k, \theta) \big].
+\end{aligned}
+$$
+
+Where we have used the standard Gamma integration trick of rewriting
+$$
+{z z^{k-1} \over \Gamma(k) \theta^k} = {k \theta z^k \over\Gamma(k+1) \theta^{k+1} }.
+$$
+
+In the special case of the equal event window then the discrete delay distribution is:
+
+$$
+\begin{aligned}
+f_n &= (n+2) F_T(n+2; k, \theta) + n F_T(n; k, \theta) - 2  (n+1) F_T(n+1; k, \theta) \\
+ &+  k \theta \Big( 2 F_T(n+1; k+1, \theta) - F_T(n; k+1, \theta) - F_T(n+2; k+1,\theta)  \Big).
+\end{aligned}
+$$
+
+Which was also found by Cori _et al_ [@cori2013new].
+
+#### Log-Normal distribution
+
+The Log-Normal distribution has the density function:
+
+$$
+f_T(z;\mu, \sigma) = {1 \over z \sigma \sqrt{2\pi}} \exp\left( - {(\log(z) - \mu)^2 \over 2 \sigma^2} \right).
+$$
+And distribution function:
+
+$$
+F_T(z;\mu, \sigma) = \Phi\left( {\log(z) - \mu \over \sigma} \right).
+$$
+Where $\Phi$ is the standard normal distribution function.
+
+This gives a solution to the survival function of $S_+$ in terms of analytically available functions:
+
+$$
+\begin{aligned}
+Q_{S+}(t ;\mu, \sigma) &= Q_T(t + W_P;\mu, \sigma) + { 1 \over W_P} \int_t^{t+W_P} {1 \over z \sigma \sqrt{2\pi}} \exp\left( - {(\log(z) - \mu)^2 \over 2 \sigma^2} \right) (z-t)~ dz, \\
+&= Q_T(t + W_P;\mu, \sigma) + { 1 \over W_P} \Big[ e^{\mu + \frac{1}{2} \sigma^2} \Delta_{W_P}F_T(t; \mu + \sigma^2, \sigma) - t\Delta_{W_P}F_T(t; \mu, \sigma) \Big]
+\end{aligned}
+$$
+
+Where we have used the Log-Normal integration trick of making a further substitution $y = (\ln z - \mu) / \sigma$ and using the shift property of the standard normal distribution function. This gives:
```

Comment:
nice. cite

Reply:
also subbing into to equation blah blah blah

Reply:
Yeah, I don't actually know a citation for this. I put "The log-normal integration trick" as a bit of a placeholder. I'll try and find a ref, its pretty obvs once you spot it.